config/default.js

```diff
@@ -110,6 +110,7 @@ module.exports = {
     'enableDevTools',
     'enableFeatureExperienceSurvey',
     'enableFeatureHomeHeroGuides',
+    'enableFeatureStaticThemesForAndroid',
```

Comment:
maybe it's just me but I find this name a little confusing since we currently are not displaying static themes on Android, yet you have "true" here…

maybe it can be renamed to just "enableFeatureThemesForAndroid" ?

Reply:
That sounds correct to me, the feature is about enabling static themes for Android, which is the case by default but not in -dev anymore. When we'll push that to production, the default config will be `false`.

Reply:
oh ok..I didn't realize static themes were on android on stage etc. I was looking at dev and saw they weren't there.

Reply:
Ok, I will add a comment then.

Reply:
thank you :)